docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/apply-simplified-microservice-cqrs-ddd-patterns.md

```diff
@@ -29,6 +29,11 @@ An example of this kind of service is the ordering microservice from the eShopOn
 
 The application layer can be the Web API itself. The important design aspect here is that the microservice has split the queries and ViewModels (data models especially created for the client applications) from the commands, domain model, and transactions following the CQRS pattern. This approach keeps the queries independent from restrictions and constraints coming from DDD patterns that only make sense for transactions and updates, as explained in later sections.
 
+## Additional resources
+
+ - **Greg Young. Versioning in an Event Sourced System** (Free online e-book) \
```

Comment:
I see a minimum price of $5? I don't think we should be linking to paid resources,

Reply:
The book is free to read online, anyway I checked with @CESARDELATORRE first.

![image](https://user-images.githubusercontent.com/7357806/56505625-06d6c300-6514-11e9-923e-504a7a4ea4c0.png)

Perhaps changing it to "Free to read online e-book"?

Reply:
It's easy to miss that. How about you keep the Free online e-book but link to https://leanpub.com/esversioning/read instead?

Reply:
Yep, certainly much better 😊